common/SmokeTests/SmokeTest/Program.cs

```diff
@@ -22,6 +22,15 @@ static async Task<int> Main(string[] args)
             // https://github.com/Azure/azure-sdk-for-net/issues/11297
             // await CosmosDBTest.RunTests();
 
+            Console.WriteLine();
```

Comment:
It would be nice if we didn't need to commit this boiler plate code for each sample we include in our smoke tests.

Reply:
I'll defer to @danieljurek on that.   I certainly agree with the perspective, but the goal of creating a robust infrastructure for smoke tests orthogonal to my immediate goal of ensuring test coverage on this single sample.   I'm happy to help with the efforts, but I'll need to block additional time at a later point to do so.